Mati final

matias-araya-02 opened the pull request:
Va queriendo, falta arreglar unas cosas pero asi ya voy viendo la parte de categorias ;)

matias-araya-02 committed:
Feat: Mejora en la visualización de artículos eliminados y activos en la tabla

matias-araya-02 committed:
Feat: Mejora en los botones de editar y eliminar en el formulario de artículos manufacturados

matias-araya-02 committed:
Feat: Agregar modal para editar la cantidad de insumos en el formulario de artículos manufacturados

matias-araya-02 committed:
Feat: centralizar botones y título en el modal de artículo manufacturado

- El botón "+ Agregar Insumo" ahora se muestra debajo de "Tiempo Estimado" en la columna izquierda.
- Los botones de acción (Cancelar, Guardar, Cerrar) están centrados en la parte inferior del modal.
- El título del modal ahora aparece centrado.
- Mejora de la experiencia visual y consistencia en la interfaz de gestión de artículos manufacturados.

matias-araya-02 committed:
Feat: mejorar estilos de botones y ajustar el modal de selección de insumos

- Se achicó el ancho del modal (max-w-md).
  - Se centralizaron los botones de acción.
  - El botón "Agregar Insumo" ahora usa el color rojo primary.

matias-araya-02 committed:
Feat: actualizar enlaces en el menú de navegación y mejorar estilos en modales

matias-araya-02 committed:
Feat: agregar lógica de eliminación lógica y restauración en el formulario de artículos manufacturados

matias-araya-02 committed:
Feat: agregar gestión de insumos con formularios y lógica de estado

matias-araya-02 committed:
Feat: agregar campo de Precio Venta en el formulario de insumos y en la tabla de insumos

matias-araya-02 committed:
Mejoras en modales de insumo y manufacturados + store

- Se centralizó y mejoró la visibilidad del checkbox "Habilitado" en el modal de artículos manufacturados (modo view).
- Se ajustó el modal de insumos para mostrar campos requeridos, checkboxes ("Habilitado" y "Es para elaborar") y "precioVenta" en tabla y formulario.
- Se mejoró la lógica de soft delete y sincronización de estado en insumoStore.
- Ajustes visuales y de usabilidad en ambos modales para mayor coherencia y experiencia de usuario.

LautaroEmanuelG committed:
Merge branch 'develop' into Mati_Final

LautaroEmanuelG commented:
No olvides actualizar desde el main la rama, cuando este terminada la PR decime asi borro la rama esta despues de aceptar la PR y empezas a trabajar con features

LautaroEmanuelG commented on src/modules/HU23_CRUDInsumos/components/ModalInsumoForm.tsx:
```diff
@@ -0,0 +1,269 @@
+  if (!open) return null;
+
+  return (
+    <div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50">
```
Aqui en vez de black usar "negro"

LautaroEmanuelG commented on src/modules/HU23_CRUDInsumos/components/ModalInsumoForm.tsx:
```diff
@@ -0,0 +1,269 @@
+
+  return (
+    <div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50">
+      <div className="bg-white p-6 rounded-lg shadow-lg w-full max-w-md">
```
Igual en vez de white "blanco"

LautaroEmanuelG commented on src/modules/HU23_CRUDInsumos/components/ModalInsumoForm.tsx:
```diff
@@ -0,0 +1,269 @@
+              Cancelar
+            </button>
+            {mode !== 'view' && (
+              <button type="submit" className="bg-primary text-white px-4 py-2 rounded">
```
"blanco"

matias-araya-02 committed:
Refactor: aplicar colores personalizados de Tailwind en modal de insumos

- Se reemplazaron los colores bg-black y bg-white por bg-negro y bg-blanco definidos en tailwind.config.js.
- Se actualizó el color del texto de los botones principales a text-blanco.
- El modal ahora respeta la paleta personalizada del proyecto para fondo, contenido y botones.

matias-araya-02 committed:
feat: mover y mejorar carga/preview de imagen en modal de artículos manufacturados

- La carga y previsualización de imagen ahora se muestra centrada y debajo de las dos columnas principales del modal (en vez de la columna izquierda).
- Se eliminó temporalmente la lógica de subida a Cloudinary; ahora solo se previsualiza la imagen seleccionada localmente.
- Se mantiene la visualización de la imagen en modo view, también centrada y debajo de las columnas.

LautaroEmanuelG committed:
Update ScreenInsumos.tsx
